.dockerignore

```diff
@@ -1,2 +1,6 @@
 .git
-autotest
\ No newline at end of file
+autotest
+build
+dist
+documentation
```

Comment:
This looks dangerous as this path is part of the repository isn't it?

Reply:
Sure, but these directories are not used in the docker image build process.